libs/novui/src/tokens/semanticSizes.tokens.ts

```diff
@@ -15,6 +15,28 @@ export const SEMANTIC_SIZES_TOKENS = defineSemanticTokens.sizes({
       },
     },
   },
+  modal: {
+    width: {
+      DEFAULT: {
+        value: '800px',
+        type: 'sizes',
+      },
+      max: {
+        value: '800px',
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** Do we want both of these values to be the same and do we need both?‏

Reply:
We can have both values but different. Currently these values are kept same for simplicity. Later I wil handle modal size